.github/workflows/conda-envs/test-compatible_env.yaml

```diff
@@ -0,0 +1,11 @@
+name: test
+channels:
+  - conda-forge
+
+dependencies:
+  - tbb
+  - tbb-devel
+  - ninja
+  - pip
+  - pip:
+    - -r requirements-test-compatible.txt
```

Comment:
I'm surprised this works. conda-forge provides packages with a specific ABI that is not the same as PyPI's manylinux. Prefer the conda-forge provided packages and only fall back on pip installations when absolutely necessary. In such cases, consider contributing the package to the conda-forge ecosystem.

Reply:
`run_tests.yml` has to install the test dependencies plus tbb, but `build_wheels.yml` has to install the test dependencies without tbb. I could not think of any other way to have two separate sets of requirements files without writing the same pinned dependencies in two different places.